test/libsolidity/StandardCompiler.cpp

```diff
@@ -74,6 +74,23 @@ bool containsAtMostWarnings(Json::Value const& _compilerResult)
 	return true;
 }
 
+string bytecodeSansMetadata(string const& _bytecode)
+{
+	/// The metadata hash takes up 43 bytes (or 86 characters in hex)
+	/// /a165627a7a72305820([0-9a-f]{64})0029$/
```

Comment:
The code was fairly compact, unfortunately the libc on travis doesn't support regex:

```
std::smatch m;
if (std::regex_match(_bytecode, m, std::regex("^(.+)a165627a7a72305820[0-9a-f]{64}0029$")))
    return m[1];
return _bytecode;
```